templates/pages/lab-content.handlebars

```diff
@@ -25,7 +25,7 @@
     {{> header }}
     <div class="container-fluid flex-fill d-flex flex-column vlabs-page-main">
       {{> breadcrumbs }}
-      <div class="row flex-grow-1 px-4 d-flex flex-nowrap">
+      <div class="row flex-grow-1 px-4 d-flex flex-nowrap flex-column flex-lg-row">
```

Comment:
I am a little uncomfortable using the row and column types for the same element. Is this use idiomatic?

Reply:
Here it makes column as the default direction for flex, and gives precedence for row direction when the display is large(lg). I have tested precedence thing, and it was working.